specs/altair/p2p-interface.md

```diff
@@ -83,18 +83,26 @@ See the [state transition document](./beacon-chain.md#beaconblockbody) for Altai
  
 This topic is used to propagate partially aggregated sync committee signatures to be included in future blocks.
 
-The following validations MUST pass before forwarding the `signed_contribution_and_proof` on the network; define `contribution_and_proof = signed_contribution_and_proof.message` and `contribution = contribution_and_proof.contribution` for convenience:
+The following validations MUST pass before forwarding the `signed_contribution_and_proof` on the network; define `contribution_and_proof = signed_contribution_and_proof.message`, `contribution = contribution_and_proof.contribution`, and the following function `get_sync_subcommittee_pubkeys` for convenience:
+
+```python
+SYNC_SUBCOMMITTEE_SIZE = SYNC_COMMITTEE_SIZE // SYNC_COMMITTEE_SUBNET_COUNT
+
+def get_sync_subcommittee_pubkeys(state: BeaconState, subcommittee_index: uint64) -> Sequence[BLSPubkey]:
```

Comment:
Does p2p spec get built with python? I don't suppose so

Either toss this in validator.md or add p2p-interface.md into the build

Reply:
makes more sense to scan `p2p-interface.md` but looks like we dont currently support constants in markdown blocks so had to fold it in